src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -1344,6 +1344,10 @@ namespace CondenserLoopTowers {
                 ErrorsFound = true;
             }
 
+            // set tower design water outlet and inlet temperatures
+            tower.DesOutletWaterTemp = tower.DesignInletWB + tower.DesignApproach;
+            tower.DesInletWaterTemp = tower.DesOutletWaterTemp + tower.DesignRange;
```

Comment:
These two variables were properly calculated for all CT Types except CoolingTower:VariableSpeed.

This fixes the "Leaving Water Setpoint Temperature" displaying as zero